agent/functional_tests/tests/functionaltests_unix_test.go

```diff
@@ -962,3 +962,104 @@ func TestTwoTasksSharedLocalVolume(t *testing.T) {
 	rExitCode, _ := rTask.ContainerExitcode("task-shared-vol-read")
 	assert.Equal(t, 42, rExitCode, fmt.Sprintf("Expected exit code of 42; got %d", rExitCode))
 }
+
+// TestHostPIDNamespaceSharing tests the visibility of an executable running on one Task from
+// another Task. Both tasks share their PID namespace with Host. Second Task should see the
+// running executable.
+func TestHostPIDNamespaceSharing(t *testing.T) {
+	agent := RunAgent(t, nil)
+	defer agent.Cleanup()
+	agent.RequireVersion(">=1.20.0")
```

Comment:
can you add a comment to change this version?

Reply:
I'll update these tests to require version >=1.20.3 because that's the current version of Agent in which we support the pidMode and ipcMode. I don't think this number needs to be updated in the future.

Reply:
shouldn't it be the version in which your changes are released?